stm/stm.ml

```diff
@@ -1683,12 +1683,12 @@ end = struct (* {{{ *)
     | `OK name ->
         let con = Nametab.locate_constant (Libnames.qualid_of_ident name) in
         let c = Global.lookup_constant con in
-        let o = match c.Declarations.const_body with
+        let _o = match c.Declarations.const_body with
```

Comment:
Why not remove this now useless match?

Reply:
It's still checking the assertion though.

Reply:
Is the assert a check or just because we have to put something in that branch?
If it's a check then change the match to return unit.

Reply:
I don't know what was the original intent, but probably both. I'll turn it into a unit value then.